### DISCUSSION TITLE
Adding new token into my my light wallet

### DISCUSSION THREAD

**Discussion by mycorrhizae82** [Chia-Network/chia-blockchain#10128] [Support]
Hello. I'm a fairly non technical person but I am really excited about the Chia project. This is also my first time using GitHub. If someone could help me out breaking things down like I am 8 years old I would really appreciate it.

My goal is to purchase XCH using these instructions: https://www.chia.net/buy-xch/

My problem is that the Chia software isn't giving me an option to ADD TOKEN as was shown in the YouTube tutorial. 
<img width="959" alt="No ADD TOKEN" src="https://user-images.githubusercontent.com/99140642/152696198-1acf279a-2118-4a42-b0d7-99dcd2540653.png">

Why isn't my software showing the ADD TOKEN option and how do I get it fixed?

Thanks!

**Comment by esaung** [comment]
Hi @mycorrhizae82 
It looks like you are currently using our 1.2.11 full node client. You will need to download and use our light wallet standalone app. You can get it from our downloads page: www.chia.net/downloads

scroll down until you see this section, and then download for your appropriate system. (heed the warning if you are also trying to farm, but if not, then you don't need to worry about it).
<img width="498" alt="Screen Shot 2022-02-07 at 1 40 09 PM" src="https://user-images.githubusercontent.com/93196638/152859695-9d22d6a0-1f72-49cc-8a43-7b4b448cc0f5.png">

In the light wallet app you will the "Add token" option.

Thanks,
E-ming

**Reply by mycorrhizae82** [reply]
Ah, thank you @esaung . When I first downloaded the software I actually attempted to farm, but I think my equipment didn't have enough space. I am only using a Macbook pro and I had an old Western Digital external hard drive with a single terrabite worth of space. Is it still possible and/or worth it to farm with this?

**Reply by mycorrhizae82** [reply]
I have another question I just thought of....If I end up downloading the light wallet app should I completely delete the Chia full node software I have?

**Comment by esaung** [comment]
hi @mycorrhizae82 ,
Possible? yes, worth it? It will help you learn when you get it setup and go from there. If you haven't found this, try starting here if you want to set up your farm: https://github.com/Chia-Network/chia-blockchain/wiki

if you download the light wallet, you will install the app over the Chia full node. not all the files of the chia full node software will get overwritten. if you look in ~/.chia you will see a "mainnet" folder and "standalone_wallet" folder. So if you do have your farming setup, you just won't be farming with the the light wallet app, but you can always recover back to your farming setup by installing the full node. So please be aware of that if you decide to do that (heed the warning on the download for the light wallet).

**Reply by mycorrhizae82** [reply]
Thank you @esaung. I think for the time being I just want to try farming and deal with the light wallet stuff later. I've read through the Github Beginners guide page to get started. Right now I have a full node working and it is synced, so it looks like I need to plot next. My current laptop only has about 134 GB of storage, and I read somewhere in Github that I needs at least 250 GB to get the plots set up. I do have a Western Digital external hardrive with 1 terrabite of storage. I'd like to use this to put the plots on and farm.  

Will this work? 
When I tell the Chia app that I want to set up plots how do I make sure they go onto my external hardrive and not my computer?
Thanks

**Comment by esaung** [comment]
@mycorrhizae82 - yes you will need to use the external hard drive. when you setup your plots you will be asked to specify a temp drive and the final location where the plots will be stored. Because you only have 134gb of space on your laptop drive, you'll need to specify a folder on your external hard drive. You should consider setting up your plots to contribute to a pool since you are starting with a 1TB drive. instructions are here: https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide

I also suggest reading through the "Plotting basics" to understand what settings to set when starting the plotting process.

**Reply by mycorrhizae82** [reply]
Thank you @esaung. I decided to get the light wallet download on my computer to use with Stably. I expect to be getting an additional laptop soon. I will use the new laptop for plotting and farming. For now I'm good! 👍
